### ISSUE TITLE
Give the possibility to finish comments later

### ISSUE THREAD

**Issue by ericlebihan** [lesintegristes/lesintegristes-theme#30]
Some of our readers use to let long comments, why not let them the possibility to begin a comments and finish it later ? Any possibilities you know ? Local storage ?

**Comment by kaneel** [comment]
LocalStorage is possible but... I have doubts about it, I prefer first
throws over deep thoughts. A comment box should remain for adding his 2
bucks only.
If the user want to add his 1000 bucks, he could blog about it and mention
his post as an answer to ours.

2012/9/27 ericlebihan notifications@github.com

> Some of our readers use to let long comments, why not let them the
> possibility to begin a comments and finish it later ? Any possibilities you
> know ? Local storage ?
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/lesintegristes/lesintegristes-theme/issues/30.

## 

// Guillaume *kaneel *Richard http://mynameiskaneel.com
// With many lower than 3

**Comment by ericlebihan** [comment]
Do you want the readers of les intégristes to write an article when they exceed the length that a comments should have ? Do we have to restrain the number of characters of a comment ?

**Comment by bpierre** [comment]
Do you mean the possibility to edit published comments, or to save a comment draft?

The problem is that we have to save it on the browser (cookie or local storage) commenters have no accounts. It seems hard to provide a solution to start a comment on a smartphone, and finish it on a desktop.

OR… we could use the cookie + a secret code. The secret code (short, 4 chars) fill the comment form with your comment. You could also send yourself the secret code.

![prototype](http://scri.ch/cwd.png)

**Comment by bpierre** [comment]
> Do you want the readers of les intégristes to write an article when they exceed the length that a comments should have ?
> Do we have to restrain the number of characters of a comment ?

Extreme, cool :-) But I think the length of a comment is not always related to its conversion to an article.

But what about displaying a link to the “Contribute” page (yes, we need to discuss this :D **– edit: #31**) when the comment exceeds a given length? The commenter could judge by itself if the content is worthy enough to be converted to an article.

I think the real problem with long comments is they are not sufficiently highlighted. Currently, comments have a smaller font size, and are italicized. The comments need to be as readable as the article.

**Comment by bpierre** [comment]
OK forget my drawing, we dont need this shitty UI, the email is already needed to write a comment.

Updated drawing:

![prototype 2](http://scri.ch/cwe.png)

**Comment by ericlebihan** [comment]
Yes ! I would see beyond a certain number of characters a message popping :

"Hey, you've just  reached the maximum of characters allowed, are you sure you are writing a comment ? What about express your point of view in a wider way into an article for the contribution page ?"

Shall we have a mention "related to the article :link of the article" ?

**Comment by bpierre** [comment]
> "Hey, you've just reached the maximum of characters allowed, are you sure you are writing a comment ? What about express your point of view in a wider way into an article for the contribution page ?"

Yes, and a button, and when you click on it, you are redirected on the contribution page, with the form pre-filled.

> Shall we have a mention "related to the article :link of the article" ?

We could just use trackbacks? And highlight internal trackbacks?

**Comment by ericlebihan** [comment]
<blockquote>
Yes, and a button, and when you click on it, you are redirected on the contribution page, with the form pre-filled.
</blockquote>

That's what I had in mind :-)

Ok for using trackbacks

**Comment by gregoiredierendonck** [comment]
> Do we have to restrain the number of characters of a comment ?

I think it's a strange idea... If someone want to speak, what's wrong? :) Not good for me.

> I think the real problem with long comments is they are not sufficiently highlighted. Currently, comments have a smaller font size, and are italicized. The comments need to be as readable as the article.

I will work on it for the new design... It's a really big priority!

What about having a preview of the comments before posting, like in GitHub? I think it's a really good thing.

**Comment by bpierre** [comment]
> What about having a preview of the comments before posting, like in GitHub? I think it's a really good thing.

Yes, and Markdown => #32
